common/src/main/java/earth/terrarium/heracles/client/screens/AbstractQuestScreen.java

```diff
@@ -77,13 +79,21 @@ public void render(@NotNull GuiGraphics graphics, int i, int j, float f) {
     }
 
     protected void renderBg(GuiGraphics graphics, float partialTick, int mouseX, int mouseY) {
-        graphics.fill(0, 0, width, height, 0xD0000000);
-        graphics.blitRepeating(HEADING, 0, 0, this.width, 15, 0, 0, 128, 15);
+        RenderSystem.enableBlend();
+        RenderSystem.defaultBlendFunc();
         if (drawSidebar()) {
             int sidebarWidth = (int) (this.width * 0.25f) - 2;
-            graphics.blitRepeating(HEADING, sidebarWidth, 15, 2, this.height - 15, 128, 0, 2, 256);
-            graphics.fill(sidebarWidth, 0, sidebarWidth + 2, 13, 0x80808080);
+            WidgetUtils.blitTiling(graphics, HEADING, 0, 15, sidebarWidth, height - 15, 0, 128, 128, 128); // Side Background
+            WidgetUtils.blitTiling(graphics, HEADING, sidebarWidth + 2, 15, width - sidebarWidth, height - 15, 128, 128, 128, 128); // Main Background
+            WidgetUtils.blitTiling(graphics, HEADING, 0, 0, sidebarWidth, 15, 0, 0, 128, 15); // Side Header
+            WidgetUtils.blitTiling(graphics, HEADING, sidebarWidth + 2, 0, width - sidebarWidth, 15, 130, 0, 126, 15); // Main Header
+            WidgetUtils.blitTiling(graphics, HEADING, sidebarWidth, 0, 2, 15, 128, 0, 2, 15); // Header Separator
+            WidgetUtils.blitTiling(graphics, HEADING, sidebarWidth, 15, 2, height - 15, 128, 15, 2, 113); // Body Separator
+        } else {
+            WidgetUtils.blitTiling(graphics, HEADING, 15, 0, width, height - 15, 128, 128, 128, 128); // Main Background
```

Comment:
15 down, not 15 right!

```suggestion
            WidgetUtils.blitTiling(graphics, HEADING, 0, 15, width, height - 15, 128, 128, 128, 128); // Main Background
```